api4/emoji_test.go

```diff
@@ -5,8 +5,11 @@ package api4
 
 import (
 	"bytes"
+	"github.com/mattermost/mattermost-server/v5/utils/fileutils"
 	"image"
 	_ "image/gif"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion
	"bytes"
	"image"
	_ "image/gif"
	"os"
	"path/filepath"
	"testing"
	
	"github.com/mattermost/mattermost-server/v5/utils/fileutils"

	"github.com/stretchr/testify/assert"
```

We tend to separate imports in three different groups. Standard library, mattermost packages and external dependencies.

Reply:
do you use gofmt/goimports? because that kind of thing is usually handled by such tools

Reply:
done

Reply:
> do you use gofmt/goimports? because that kind of thing is usually handled by such tools

We do use `gofmt` as part of [golangci-lint](https://github.com/golangci/golangci-lint). We have a `make check-style` command to verify that files are formatted correctly. That said, we do not enforce the grouping of imports.